### DISCUSSION TITLE
prefill create page from query params

### DISCUSSION THREAD

**Discussion by ralyodio** [dailydotdev/daily#1057] [Feature Requests]
Want to do smething like this: https://app.daily.dev/squads/create?title=foo&body=bar&link=http...

**Comment by idoshamun** [comment]
Hi @ralyodio,

That's a great idea!

**Comment by ralyodio** [comment]
thanks. i'll submit a lot more links.

**Comment by rebelchris** [comment]
@ralyodio Been a while, but we shipped this now :) 
title, body, link, plus existing share/poll precedence
